src/main/java/com/ericsson/ei/controller/QueryMissedNotificationControllerImpl.java

```diff
@@ -46,21 +50,34 @@
      * @param subscriptionName
      * @return ResponseEntity
      */
-    public ResponseEntity<QueryResponse> getQueryMissedNotifications(@RequestParam("SubscriptionName") final String subscriptionName) {
+    @Override
+    public ResponseEntity<QueryResponse> getQueryMissedNotifications(
+            @RequestParam("SubscriptionName") final String subscriptionName) {
+        ObjectMapper mapper = new ObjectMapper();
         QueryResponse queryResponse = new QueryResponse();
+        QueryResponseEntity queryResponseEntity = new QueryResponseEntity();
         try {
             List<String> response = processMissedNotification.processQueryMissedNotification(subscriptionName);
-            queryResponse.setResponseEntity(response.toString());
+            JsonNode responseJson = null;
```

Comment:
this needs to be done in the same as in QueryAggregatedObjectControllerImpl.java.

This means that you should map the string directly into class and not add an extra property called missedNotification to the response.

Reply:
done